### PR TITLE
Completed US12 TC002-done

### PR DESCRIPTION
**Pull request by kgasimov**
Finished by adding some cucumber staff here

### CODE REVIEW

File: src/test/java/day2_team_colaboration/TC002.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package day2_team_colaboration;
+
+public class TC002 {
+    // Pull request from IJ
```
Comment (SDET222) [inline src/test/java/day2_team_colaboration/TC002.java:4]: what means IJ?

**Review by SDET222 — CHANGES_REQUESTED** [review]
could you please explain?

File: src/test/java/day2_team_colaboration/TC002.java (hunk 1 above)
Comment (kgasimov) [reply src/test/java/day2_team_colaboration/TC002.java:4]: IJ stands for intelli J

Comment (SDET222) [reply src/test/java/day2_team_colaboration/TC002.java:4]: good

**Review by SDET222 — APPROVED** [review]
approved, good job

**Pull request merged**